final-project/scripts/main.js

```diff
@@ -1,10 +1,110 @@
-import "../styles/main.scss";
-
-// testing that `const` and arrow functions transpile correctly
-const before = "Hello -world!";
-const after = before
-  .split("")
-  .filter(char => char !== "-")
-  .join("");
-console.log({ before });
-console.log({ after });
+import "../styles/main.scss";
+import Axios from "axios";
+
+function showPage() {
+  document.getElementById("body-layout").classList.toggle("layout-wrapper--loading");
+  document.getElementById("page-loader").style.display = "none";
+}
+
+window.onload = function () {
+  //Wait just to simulate a low connection and a delay
+  setTimeout(showPage, 3000);

```

Comment:
probably you forgot to remove this, also as mentioned in the debugging class you can simulate this with the browser dev tools